### PR TITLE
Fix metadata tab

### PR DESCRIPTION
**Pull request by marcinbodnar**

### CODE REVIEW

File: src/WidgetMetadataEditor.jsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ if (metadata === null) {
 return (
   <Widget
     key={widgetPath}
-    src="${REPL_MOB}/widget/MetadataEditor"
+    src="${REPL_ACCOUNT}/widget/MetadataEditor"
```
Comment (shelegdmitriy) [inline src/WidgetMetadataEditor.jsx:13]: @marcinbodnar is this was just a typo?

Comment (marcinbodnar) [reply src/WidgetMetadataEditor.jsx:13]: What do you mean?

Comment (marcinbodnar) [reply src/WidgetMetadataEditor.jsx:13]: `near/widget/MetadataEditor` is the proper metadata editor widget - `mob.near/widget/MetadataEditor` is not working properly.

Comment (shelegdmitriy) [reply src/WidgetMetadataEditor.jsx:13]: Got it, thank you!

**Review by shelegdmitriy — APPROVED** [review]

**Pull request merged**